pkg/bff/api/v1/client_test.go

```diff
@@ -596,6 +597,42 @@ func TestMemberDetails(t *testing.T) {
 	require.Equal(t, fixture, out)
 }
 
+func TestAttention(t *testing.T) {
+	fixture := &api.AttentionReply{
+		Messages: []*api.AttentionMessage{
+			{
+				Message:  bff.SubmitMainnet,
+				Severity: models.AttentionSeverity_INFO,
+				Action:   models.AttentionAction_SUBMIT_MAINNET,
+			},
+			{
+				Message:  fmt.Sprintf(bff.CertificateRevoked, "testnet"),
+				Severity: models.AttentionSeverity_ALERT,
+				Action:   models.AttentionAction_CONTACT_SUPPORT,
+			},
+		},
+	}
+
+	// Create a Test Server
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		require.Equal(t, http.MethodGet, r.Method)
+		require.Equal(t, "/v1/attention", r.URL.Path)
+
+		w.Header().Add("Content-Type", "application/json; charset=utf-8")
+		w.WriteHeader(http.StatusOK)
+		json.NewEncoder(w).Encode(fixture)
+	}))
```

Comment:
I would also be interested in having a test where the server returned 204 since that is a new addition to the code base.

Reply:
Sure, I can add that.